ee/debug/checkups/performance.go

```diff
@@ -8,9 +8,15 @@ import (
 	"github.com/kolide/launcher/ee/performance"
 )
 
+const (
+	cpuPercentThreshold = 30
+	memPercentThreshold = 30
```

Comment:
I don't think memory percentage is a meaningful number. We should pick some numbers for go and non-go

Reply:
Took a quick peek at our metrics -- how about over 200MB for go, over 100MB for non-go?

Reply:
Went with those, can adjust as needed!

Reply:
My guess is we'll see a lot of errors here, but I'm game to start with those numbers.